franklin/builder/models.py

```diff
@@ -47,12 +47,14 @@ def save(self, *args, **kwargs):
         base_path = os.environ['BASE_PROJECT_PATH']
         self.path = "{0}/{1}".format(base_path, self.repo_name)
         if not self.url:
+            repo_name = self.repo_name.split("/")[1]
             self.url = "{0}.{1}".format(
-                self.repo_name.replace('/', '-'),
+                repo_name,
                 os.environ['BASE_URL']
             )
         super(Site, self).save(*args, **kwargs)
-        self.build()
+        if os.environ['ENV'] is 'production':
```

Comment:
perhaps `os.environ['ENV'] is not 'test'` and we set the environment to be `test`? We will need a way to run things end to end outside of just prod.